frontend/src/metabase/nav/components/PaymentBanner/PaymentBanner.tsx

```diff
@@ -0,0 +1,54 @@
+import { jt, t } from "ttag";
+import Banner from "metabase/components/Banner";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import MetabaseSettings from "metabase/lib/settings";
+
+interface PaymentBannerProps {
+  isAdmin: boolean;
+  tokenStatusStatus: string | undefined;
+}
+
+export const PaymentBanner = ({
```

Comment:
I don't see tests for this component. Do they exist?

Reply:
They exist. Here are the tests https://github.com/metabase/metabase/blob/master/e2e/test/scenarios/admin/subscription/payment-failure.cy.spec.js.

[Here's another related comment](https://github.com/metabase/metabase/pull/31210#discussion_r1222738533), since this component used to be named `AppBanner`

Reply:
Added tests